drivers/afe/ad4110/ad4110.h

```diff
@@ -142,6 +142,18 @@
 /*************************** Types Declarations *******************************/
 /******************************************************************************/
 
+/* If AD4110_AFE_ADC_CLOCKED selected, select AD4110_ADC_INT_CLK_CLKIO */
+enum ad4110_adc_clk_sel {
+	AD4110_ADC_INT_CLK,
+	AD4110_ADC_INT_CLK_CLKIO,
+	AD4110_ADC_EXT_CLK,
+};
+
+enum ad4110_afe_clk_cfg {
+	AD4110_AFE_INT_CLOCK,
+	AD4110_AFE_ADC_CLOCKED = 2
```

Comment:
if you explicitly initialize `AD4110_AFE_ADC_CLOCKED` enum member, than do it also for `AD4110_AFE_INT_CLOCK`